e2e/cephfs.go

```diff
@@ -126,14 +129,53 @@ var _ = Describe("cephfs", func() {
 					Fail(err.Error())
 				}
 			}
+			if upgradeTesting {
+				// fetch current working directory to switch back
+				// when we are done upgrading.
+				cwd, err := os.Getwd()
+				if err != nil {
+					Fail(err.Error())
+				}
+				err = upgradeAndDeployCSI(upgradeVersion, "cephfs")
+				if err != nil {
+					Fail(err.Error())
+				}
+				createConfigMap(cephfsDirPath, f.ClientSet, f)
+				createCephfsSecret(f.ClientSet, f)
+				createCephfsStorageClass(f.ClientSet, f, true, "")
+				pvcPath := cephfsExamplePath + "pvc.yaml"
+				appPath := cephfsExamplePath + "pod.yaml"
+				_, app := createPVCAndAppBinding(pvcPath, appPath, f, deployTimeout)
+				err = deletePod(app.Name, app.Namespace, f.ClientSet, deployTimeout)
+				if err != nil {
+					Fail(err.Error())
+				}
+				deleteCephfsPlugin()
+
+				// support of PSP is available in current master, so we don't need to
+				// skip it now.
+				skipPSP = false
+				err = os.Chdir(cwd)
+				if err != nil {
+					Fail(err.Error())
+				}
+				deployCephfsPlugin()
+				// validate if the app gets bound to a pvc created by
+				// an earlier release.
+				err = createApp(f.ClientSet, app, deployTimeout)
+				if err != nil {
+					Fail(err.Error())
+				}
+				Skip("Skipping CephFS E2E")
```

Comment:
the skip should be at starting not at end

Reply:
Once upgrade testing is done, we don't want to run the rest of the e2e tests, due to that I have skipped the e2e after successful upgrade testing.

Reply:
if we don't want to do it, we can create a new file for upgrade testing same as rbd.go and cephfs.go and have Describe, etc.